dotcom-rendering/src/components/ArticlePage.tsx

```diff
@@ -55,6 +58,21 @@ export const ArticlePage = (props: WebProps | AppProps) => {
 				`}
 			/>
 			<SkipTo id="maincontent" label="Skip to main content" />
+			<SkipTo id="navigation" label="Skip to navigation" />
+			{article.imagesForLightbox && (
```

Comment:
Exciting stuff

On that switch, I have a commit ready to wrap both the code in `ArticlePage` and also the code that adds the `ClickOverlay` to each image on the page but, unless things have changed since I left (possible!), I think you need to add the [your experiment](https://github.com/guardian/frontend/pull/26266) to the list of ActiveExperiments ([here](https://github.com/guardian/frontend/blob/main/common/app/experiments/Experiments.scala#L7)), otherwise it won't appear in the switchboard or list of 'switches'.

Reply:
@oliverlloyd annoyingly, you're going to have to pull `main` in for the tests to pass (there was [a dependency issue](https://github.com/guardian/dotcom-rendering/pull/8133)). If this is an issue, let me know, and I'll bypass the

Reply:
It's now [up to date with main](https://github.com/guardian/dotcom-rendering/pull/7129/commits/e61f6b6590790022ad8316dd2a579e6dd3bb2ae6) and [behind a switch](https://github.com/guardian/dotcom-rendering/pull/7129/commits/460e83d8d4ed2b6e19958413d60dd74af1400d99)